app/models/request.rb

```diff
@@ -0,0 +1,14 @@
+class Request < ApplicationRecord
+  has_paper_trail
+  belongs_to :user
+  belongs_to :supervisor, foreign_key: :supervisor_id, class_name: 'User', optional: true
```

Comment:
Yes, I forgot about this, you listed it as something we wanted in the issue. Based on this, I was going to show which supervisor approved it, and only show this to other supervisors. Does this still make sense to do?

Also should it be called `approved_by` with `approved_by_id` instead of `supervisor` with `supervisor_id`?

Reply:
Yes, `approved_by` makes more sense.